http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/service/LeonardoServiceSpec.scala

```diff
@@ -1370,13 +1372,16 @@ class LeonardoServiceSpec extends TestKit(ActorSystem("leonardotest")) with Flat
     dbFutureValue { _.clusterQuery.updateClusterStatus(clusterCreateResponse.id, ClusterStatus.Stopped) }
     dbFutureValue { _.clusterQuery.updateClusterCreatedDate(clusterCreateResponse.id, new SimpleDateFormat("yyyy-MM-dd").parse("2018-12-31").toInstant) }
 
-    // start the cluster and verify exception
-    leo.startCluster(userInfo, project, name1).failed.futureValue shouldBe a [ClusterOutOfDateException]
+    // start the cluster
+    leo.startCluster(userInfo, project, name1).futureValue
 
-    // cluster status should still be Stopped in the DB
-    dbFutureValue { _.clusterQuery.getClusterByUniqueKey(clusterCreateResponse) }.get
-      .status shouldBe ClusterStatus.Stopped
+    // cluster status should Running and have new label
+    val cluster = dbFutureValue { _.clusterQuery.getClusterByUniqueKey(clusterCreateResponse) }.get
 
+    eventually {
+      cluster.status shouldBe ClusterStatus.Running
```

Comment:
I think it should be `Starting` at this point

Reply:
I was hoping to test that the cluster doesn't get stuck in Starting status, since they did when the script wasn't correctly working

Reply:
Yeah.. but the `ClusterMonitorActor` is what transitions it to `Running`, and this test is only verifying `LeonardoService`. The unit tests don't start a real cluster, so the startup script isn't even run.

We have an [automation test](https://github.com/DataBiosphere/leonardo/blob/develop/automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/notebooks/NotebookClusterMonitoringSpec.scala#L184-L221) to verify that the startup script deploys welder. I'm not sure it's worth adding another one for disabling delocalization -- it sounds tricky to set up and we can probably by confident with manual testing.